app/views/agent/partials/registrationPartial.scala.html

```diff
@@ -51,18 +51,33 @@ <h3 class="heading-small">
 </div>
 }
 
+
+@pendingDeregistration()={
+<div class="column-one-third" id="cancel-vat">
+    <h3 class="heading-small">
+        @messages("registrationPartial.pendingDeregistration.title")
+    </h3>
+    <p>@messages("registrationPartial.pendingDeregistration.content")</p>
+</div>
+}
+
 @{
     if(details.deregistration.isDefined){
         details.deregistration.flatMap(_.effectDateOfCancellation) match {
             case Some(date) => date match {
                 case future if future.isAfter(currentDate) => futureDeregistration(future)
                 case historic => historicDeregistration(historic)
-            }
-            case _ => {
+        }
+        case _ => {
                 Logger.warn("[registrationPartial][deregistrationDate] - 'deregistration' contained no 'effectDateOfCancellation'")
             }
         }
     } else {
-        cancelRegistration
+        if (details.changeIndicators.isDefined) {
```

Comment:
this if condition isn't needed - if changeIndicators isn't there then the "case _" will catch it